webflow-hacks/form-hacks/45/45.ts

```diff
@@ -0,0 +1,95 @@
+// when dom is ready
+document.addEventListener('DOMContentLoaded', function () {
+  const CHECKBOX_SELECTOR = '[fs-hacks-element="checkbox"]';
+  const SELECT_SELECTOR = '[fs-hacks-element="select"]';
+  const TOTAL_SELECTOR = '[fs-hacks-element="total-value"]';
+  const HIDDEN_INPUT_SELECTOR = '[fs-hacks-element="hidden-total"]';
+  const BRANDING_GROUP_NAME = 'Branding';
+  const DEVELOPMENT_GROUP_NAME = 'Development';
+  const checkboxes = document.querySelectorAll<HTMLInputElement>(CHECKBOX_SELECTOR);
+  const totalValueDiv = document.querySelector<HTMLDivElement>(TOTAL_SELECTOR);
+  const hiddenTotalInput = document.querySelector<HTMLInputElement>(HIDDEN_INPUT_SELECTOR);
+  const selects = document.querySelectorAll<HTMLSelectElement>(SELECT_SELECTOR);
+  const radios = document.querySelectorAll<HTMLInputElement>(
+    `input[name="${BRANDING_GROUP_NAME}"], input[name="${DEVELOPMENT_GROUP_NAME}"]`
+  );
+  if (checkboxes.length === 0 || !totalValueDiv || !hiddenTotalInput) return;
+
+  let sum = 0;
+
+  // add checkboxed value to sum
+  for (const checkbox of checkboxes) {
+    const amountToBeAdded = Number(checkbox.getAttribute('add-value'));
```

Comment:
Could the `add-value` attribute be set as a constant too? This way it's easier to understand that the checkboxes require having this attribute.
Also, I would follow the same naming convention as the other attributes, using a `fs-hacks` prefix.